Dockerfile

```diff
@@ -117,7 +117,7 @@ RUN if [ "$include_dagmc" = "true" ] ; \
 RUN if [ "$include_dagmc" = "true" ] ; \
     then mkdir DAGMC ; \
     cd DAGMC ; \
-    git clone -b develop https://github.com/svalinn/dagmc ; \
+    git clone -b develop https://github.com/svalinn/DAGMC.git ; \
     mkdir build ; \
     cd build ; \
     cmake ../dagmc -DBUILD_TALLY=ON \
```

Comment:
```suggestion
    cmake ../DAGMC -DBUILD_TALLY=ON \
```

Reply:
https://github.com/openmc-dev/openmc/blob/c262407af5eac38768910f4df7410ed08ae6ddda/Dockerfile#L129

@Shimwell We have to change this to `rm -rf /DAGMC/DAGMC /DAGMC/build ; \`

Reply:
Yep nice suggestions, the docs update would be great. We don't actually have a new dockerfile build using the GitHub actions on the main branch quite yet but we should do this suggested update to the docs before the next release.

Reply:
we might have to change 
```cmake ../dagmc ```
to
```cmake ../DAGMC ```